client/src/components/Home/Home.spec.js

```diff
@@ -9,7 +9,7 @@ describe('Home', () => {
     expect(text1).toBeInTheDocument()
 
     const text2 = screen.getByText(
-      'On this app you will have the option to rate movies in a more accurate way then on other rating forms.'
+      'On this app you will have the option to rate movies in a more accurate way than on other rating forms.'
```

Comment:
maybe instead of `[...] on other rating forms.` you could say `[...] on other rating apps`, as you're speaking of your app as an 'app' and not of a form.

Reply:
You're right with that. It's currently a placeholder text.